lib/views/admin_portal/admin_portal.dart

```diff
@@ -11,6 +12,10 @@ class AdminPortal extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(actions: [
+        IconButton(
+          onPressed: () => navigator.push(const ProfilesScreen()),
+          icon: const Icon(Icons.person),
+        ),
```

Comment:
I added a button to the Participant and Admin home screens that links to the profile page:
![profile button](https://github.com/theheartcenter-one/thc/assets/10457200/4ad314fc-7ebd-4713-911b-805264851367)

We can access the profile screen using `navigator.push()`, and Flutter will automatically add a "go back" button to the AppBar.